src/content/about/metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Metrics
+weight: 4
+---
+
+- [Worker Script Metrics](#worker-script-metrics)
+  - [Invocation statuses](#invocation-statuses)
+  - [Requests](#requests)
+  - [CPU time](#cpu-time)
+- [Zone Metrics](#zone-metrics)
+  - [Total requests](#total-requests)
+  - [Status codes](#status-codes)
+  - [Bandwidth](#bandwidth)
+  - [Subrequests](#subrequests)
+- [GraphQL](#graphql)
+
+Workers metrics show performance and usage of your Workers and help diagnose issues.
+
+Workers Metrics are available in Zone and Worker script contexts.
+
+## Worker Script Metrics
+
+Aggregates request data for an individual Worker script across all zones including your workers.dev subdomain. On your [Workers dashboard](https://dash.cloudflare.com/?account=workers/overview), click on any Worker to view its metrics.
+
+Worker script metrics can be inspected by time ranges within the last 30 days. The dashboard includes the charts and information described below.
+
+### Invocation statuses
+
+Worker invocation statuses are generated by the Workers runtime and indicate whether a Worker script executed successfully or failed to generate a response. Invocation statuses differ from HTTP status codes. In some cases, a Worker script invocation succeeds but does not generate a successful HTTP status because of another error encountered outside of the Workers runtime. Some invocation statuses result in a [Workers error code](/about/tips/debugging/#error-pages-generated-by-workers) being returned to the client.
```

Comment:
```suggestion
Worker invocation statuses are generated on every incoming request to the Workers runtime and indicate whether a Worker script executed successfully or failed to generate a response. Invocation statuses differ from HTTP status codes. In some cases, a Worker script invocation succeeds but does not generate a successful HTTP status because of another error encountered outside of the Workers runtime. Some invocation statuses result in a [Workers error code](/about/tips/debugging/#error-pages-generated-by-workers) being returned to the client.
```

Reply:
Hard to read change but it was
-  are generated *by the Workers runtime and *
+ are generated *on every incoming request to* the Workers runtime and

Reply:
👍